docs/UPDATING_DEPENDENCIES.md

```diff
@@ -0,0 +1,120 @@
+2. Find the version you want to update to (e.g., `1.22.0`)
+3. Check the `package.json` to identify the `playwright`, `playwright-core`, and `playwright-chromium` versions
+
+### Step 2: Create the New Playwright Branch
+
+In the [elastic/playwright](https://github.com/elastic/playwright) fork:
+
+1. Checkout the corresponding Microsoft Playwright release tag
+
+   ```bash
+   git fetch upstream
+   git checkout v1.57.0  # Replace with target version
+   ```
+
+2. Create a new branch following the naming convention `{version}-recorder`
+   ```bash
+   git checkout -b 1.57.0-recorder
+   ```
+
+### Step 3: Cherry-pick the Recorder Patches
+
+The elastic/playwright fork contains custom commits on top of the Microsoft release. You need to apply these patches to the new branch:
+
+1. Find the current recorder branch being used (check `package.json` in synthetics-recorder). In this example, the current branch is `1.56.1-recorder`.
+2. Identify the extra commits on that branch (commits after the Microsoft `v1.56.1` release tag)
+3. Cherry-pick the **first** patch commit and resolve any conflicts
+
+   ```bash
+   git cherry-pick <first-commit-sha>
+   # Resolve conflicts if any
+   git add .
+   git cherry-pick --continue
+   ```
+
+4. Cherry-pick the **second** patch commit (this typically includes the packaging script)
+   ```bash
+   git cherry-pick <second-commit-sha>
+   ```
+
+### Step 4: Build and Publish the Tarballs
+
+After cherry-picking the commits:
+
+1. Run the packaging script to create the tarballs
+2. Push the tarballs to the remote branch
+   ```bash
+   git push origin 1.57.0-recorder
+   ```
+
+### Step 5: Update synthetics-recorder
+
+Update `package.json` in this project:
+
+1. Update the `@elastic/synthetics` version:
+
+   ```json
+   "@elastic/synthetics": "=1.22.0"
+   ```
+
+2. Update the Playwright package URLs to point to the new branch:
+
+   ```json
+   "playwright": "https://github.com/elastic/playwright/raw/1.57.0-recorder/playwright.tgz",
+   "playwright-chromium": "https://github.com/elastic/playwright/raw/1.57.0-recorder/playwright-chromium.tgz",
+   "playwright-core": "https://github.com/elastic/playwright/raw/1.57.0-recorder/playwright-core.tgz"
+   ```
+
+3. Clean install the dependencies:
+
+   ```bash
+   rm -rf node_modules
+   npm install
+   ```
+
+4. Run tests locally to verify everything works:
+
+   ```bash
+   # Unit tests
+   npm run test:unit
+
+   # E2E tests (run in separate terminals)
+   npm run test:e2e:server   # Terminal 1: starts the test server
+   npm run test:e2e:runner   # Terminal 2: runs the E2E tests
+   ```
+
+### Step 6: Open a Pull Request
+
+1. Commit your changes and push to a new branch
+2. Open a PR to merge the changes into the main branch
+3. The CI pipeline will run automatically to verify all tests pass
+4. Once the pipeline is green and the PR is approved, merge the changes
+
+## Version History
```

Comment:
We should note that the person doing the upgrade needs to amend this table if we will have it here.

Reply:
I removed this part, not sure it was worth it.

https://github.com/elastic/synthetics-recorder/pull/576/commits/d336e3048e4a268be40d9802714f5fd7493b28aa